src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -111,7 +115,14 @@ public Person toModelType() throws IllegalValueException {
             throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
                     InsurancePlansManager.class.getSimpleName()));
         }
+
+        if (claims == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    Claim.class.getSimpleName()));
```

Comment:
`getSimpleName()` is a bit vague. Maybe we can try to name this more meaningfully?

Reply:
That's a built in method I think HAHA